Scss/_body.scss

```diff
@@ -0,0 +1,118 @@
+img {
+  width: 100%;
+}
+
+body {
+  font-size: 14px;
+  color: #505353;
+}
+
+ul {
+  li {
+      list-style: none;
+  }
+  margin: 0;
+  padding: 0;
+}
+
+.fw-bold {
+  font-weight: bold !important;
+}
+
+.banner-wrapper {
+  position: relative;
+  background: linear-gradient(180deg, #FFFBFB 87.5%, #E7E9EE 100%);
+}
+
+.banner-content {
+    position: absolute;
+    left: 60px;
+    top: 50%;
+    transform: translateY(-50%);
+  .banner-heading {
+    h2 {
+        font-size: 50px;
+        font-weight: normal;
+        color: #11114E;
+        margin-bottom: 0;
+        .banner-description {
+          margin-top: 10px;
+          p {
+            font-size: 20px;
+            color: #505353;
+            }
+          .banner-button {
+            margin-top: 26px;
+            button {
+              background: #B2D235;
+              box-shadow: inset 0px 4px 4px rgba(0, 0, 0, 0.25);
+              border-radius: 35px;
+              padding: 13px 67px;
+              color: #FFFFFF;
+                }
+            }
+        }
+      }
+    }
+  }
+@media (max-width: 416px) {
+  .mobile-container {
+      max-width: 280px;
+      padding: 0;
+  }
+}
+.banner-content {
+  @media (max-width: 416px) {
+      position: static;
+      transform: none;
+      height: 350px
+  }
+  .banner-heading {
+    h2 {
+      @media (max-width: 416px) {
+          font-size: 25px;
+      }
+    }
+    @media (max-width: 416px) {
+      margin-top: 10px;
+    }
+  }
+  .banner-description {
+    @media (max-width: 416px) {
+      margin-top: 16px;
+      }
+    }
+}
+.menu-navbar {
+  @media (max-width: 768px) {
+    background: #FFFBFB;
+    flex-flow: column;
+    justify-content: flex-start;
+  }
+}
+.banner-content {
+  @media (max-width: 768px) {
+    left: 26px;
+  }
+  .banner-heading {
+    h2 {
+      @media (max-width: 768px) {
+        font-size: 30px;
+        line-height: 37px;
+        }
+    }
+  }
+}
+.menu-navbar {
+    @media (max-width: 416px) {
+        background: #FFFBFB;
+        flex-flow: wrap;
+        justify-content: flex-start;
+    }
+}
+@media (max-width: 768px) {
+    .mobile-container {
+        max-width: 280px;
+        padding: 0;
+    }
+}
```

Comment:
github đang báo no newline at end of file. Cần thêm 1 dòng trống nữa vào code 
Lý do có cảnh báo này của github https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Em đã fix